src/gluonts/model/forecast.py

```diff
@@ -519,23 +540,3 @@ class Config(pydantic.BaseModel):
     num_eval_samples: int
     output_types: Set[OutputType]
     quantiles: List[str]  # FIXME: validate list elements
-
-    def process(self, forecast: Forecast) -> dict:
-        result = {}
-
-        if OutputType.mean in self.output_types:
-            result['mean'] = forecast.mean.tolist()
-
-        if OutputType.samples in self.output_types:
-            result['samples'] = (
-                forecast.samples.tolist()
-                if hasattr(forecast, 'samples')
-                else []
-            )
-
-        if OutputType.quantiles in self.output_types:
-            result['quantiles'] = [
-                forecast.quantile(q).tolist() for q in self.quantiles
-            ]
-
-        return result
```

Comment:
These changes are the same as #169, right?

Reply:
Yes, this is no longer in the latest version of the PR.